download/vs.html

```diff
@@ -1,16 +1,18 @@
 ---
 layout: base
-title: Beta Download
+title: Download - VS
 navgroup: download
-redirect_from:
-  - /beta/
-  - /Beta/
 ---
+
 <div class="row">
   <div class="small-12 columns">
-    <h1>Beta Download</h1>
+    <h1>Download</h1>
+
+    <p><b>Release channels:</b>&nbsp;<span><a href="/download/preview/">Preview</a> - <a href="/download/stable/">Stable</a> - <a href="/download/vs/">VS</a></span></p>
+
+    <p style="padding: 0 20px"><b>In the VS release channel are packages which are shipped as part of a Visual Studio for Mac release, they are the most stable but also the least frequently updated packages.</b></p>
```

Comment:
`In the VS release channel are packages` -> `The VS release channel contains packages`